template/python/Dockerfile

```diff
@@ -9,19 +11,26 @@ RUN apk --no-cache add curl \
 
 WORKDIR /root/
```

Comment:
Is this line required?

Reply:
Probably not, I think we can safely remove. Thanks Nic!